sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -44,11 +44,11 @@
 
 // DO NOT MODIFY: the path to a PEM file containing the device certificate and
 // key as well as any intermediate certificates chaining to an uploaded group certificate.
-#define ENV_DEVICE_X509_CERT_PEM_FILE_PATH "AZ_IOT_DEVICE_X509_CERT_PEM_FILE_PATH"
+#define ENV_DEVICE_X509_CERT_PEM_FILE_PATH "AZ_IOT_DEVICE_X509_CERT_PEM_FILE"
 
 // DO NOT MODIFY: the path to a PEM file containing the server trusted CA
 // This is usually not needed on Linux or Mac but needs to be set on Windows.
-#define ENV_DEVICE_X509_TRUST_PEM_FILE_PATH "AZ_IOT_DEVICE_X509_TRUST_PEM_FILE_PATH"
+#define ENV_DEVICE_X509_TRUST_PEM_FILE_PATH "AZ_IOT_DEVICE_X509_TRUST_PEM_FILE"
 
```

Comment:
I do want to update the environment name later on to include "_PATH" for consistency, but I realize that will involve updating a bunch of documentation and the generate_cert script.

Reply:
We need to be consistent. I think we should have a batch file, powershell file or bash file (or all 3) checked in as templates. That way, it's so much easier to run tests on a dev box.
In that case, we need all of these to work for all samples (I don't want to change the env variables to run telemetry vs provisioning).

Reply:
I agree with consistency, hence why I'm fine with you changing it back now.  Later, I would like to update them all to include "_PATH" because that is more precise.